### PR TITLE
README tidying

### PR DESCRIPTION
**Pull request by adam-enko**
- Apply Grazie suggestions.
- Add links to KMP and Maven Central homepages.
- Minor style fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
+* Configure the publishing plugin in your library’s project.
+* Provide your credentials to the publishing plugin so it can sign and upload your artifacts.
+* Run the publication task, either locally or using continuous integration.
 
 This guide assumes that you are:
 
 - Creating an open-source library.
 - Using macOS or Linux. If you are a Windows user, use [GnuPG or Gpg4win](https://gnupg.org/download) to generate a key pair.
-- Either not registered on Maven Central yet, or have an existing account that’s suitable for [publishing to the Central Portal](https://central.sonatype.org/publish-ea/publish-ea-guide/) (created after March 12th, 2024, or migrated to the Central Portal by their support).
+- Are either not registered on Maven Central yet, or have an existing account that’s suitable for [publishing to the Central Portal](https://central.sonatype.org/publish-ea/publish-ea-guide/) (created after March 12th, 2024, or migrated to the Central Portal by their support).
```
Comment (pahill) [inline README.md:26]: I disagree with the presumably Grazie change - the line on 24 says "you are", this change would duplicate the "are"

File: README.md [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
 
 Most of the steps here are still applicable if you’re using a different setup, but there might be some differences you need to account for. An [important limitation](https://kotlinlang.org/docs/multiplatform-publish-lib.html#host-requirements) is that Apple targets must be built on a machine with macOS.
 
-Throughout this guide, we’ll use the [https://github.com/kotlin-hands-on/fibonacci](https://github.com/kotlin-hands-on/fibonacci) repository as an example. You can refer to the code of this repository to see how the publishing setup works. Don’t forget to **replace all example values with your own** as you’re configuring your project.
+Throughout this guide, we’ll use the [https://github.com/kotlin-hands-on/fibonacci](https://github.com/kotlin-hands-on/fibonacci) repository as an example. You can refer to the code of this repository to see how the publishing setup works. You **must replace all example values with your own** as you’re configuring your project.
 
 ### Prepare accounts and credentials
 
-#### Register a namespace {#register-a-namespace}
+#### Register a namespace
```
Comment (pahill) [inline README.md:36]: There is a link to this section lower down in the doc - is this not needed?

File: README.md [hunk 3]
```diff
@@ -146,27 +145,30 @@ Run the following command to upload your public key using `gpg`, **substituting
 gpg --keyserver keyserver.ubuntu.com --send-keys F175482952A225BFC4A07A715EE6B5F76620B385CE
 ```
 
-#### Export your private key {#export-your-private-key}
```
Comment (pahill) [inline README.md:149]: Is this not needed to link to?

File: README.md [hunk 4]
```diff
@@ -226,8 +234,8 @@ mavenPublishing {
         licenses {
             license {
                 name = "The Apache License, Version 2.0"
-                url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
-                distribution = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
```
Comment (pahill) [inline README.md:237]: Great catch!

**Review by pahill — COMMENTED** [review]
Just a few items - almost there! Thank you for the help thusfar

File: README.md (hunk 2 above)
Comment (adam-enko) [reply README.md:36]: Nope :)

Maybe it's necessary in a different Markdown syntax, but for GitHub it's not needed, and [it looks strange](https://github.com/Kotlin/multiplatform-library-template?tab=readme-ov-file#register-a-namespace-register-a-namespace):

File: README.md (hunk 1 above)
Comment (adam-enko) [reply README.md:26]: Good point, I'll revert it.

Comment (adam-enko) [reply README.md:26]:
```suggestion
- Either not registered on Maven Central yet, or have an existing account that’s suitable for [publishing to the Central Portal](https://central.sonatype.org/publish-ea/publish-ea-guide/) (created after March 12th, 2024, or migrated to the Central Portal by their support).
```

**Pull request merged**